src/tests/unit/time/UtcTimeTest.php

```diff
@@ -93,12 +93,12 @@ public function testGetSecondsSinceDateTime()
             $actual = UtcTime::getSecondsSinceDateTime($testCase['value']);
             
             // Assert:
-            $this->assertEquals($testCase['expected'], $actual, sprintf(
-                'Expected %s to result in %s, not %s.',
+            $this->assertEqualsWithDelta($testCase['expected'], $actual, 1,
```

Comment:
This is poor formatting, but it's valid code, so I'm not going to hold this up. If you can't fit all the things in 120 (preferrably 80), then every argument on its own line.